[docs] Updates for Voyager 1.6.5 (#20897)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

#### Import indexes and triggers

Import indexes and triggers on the target YugabyteDB database after the `import data to target` has completed the following tasks:

- The exported snapshot has been completely imported on the target.

> - All the events accumulated in local disk by [export data from source](#export-data-from-source) during the snapshot import phase and [import data to target](#import-data-to-target) catches up in the CDC phase (you can monitor the timeline based on `Estimated Time to catch up` metric).

Reviewer: ```suggestion
- All the events accumulated on local disk by [export data from source](#export-data-from-source) during the snapshot import phase and [import data to target](#import-data-to-target) have caught up in the CDC phase (you can monitor the timeline based on `Estimated Time to catch up` metric).
```

        --post-snapshot-import true
```

> If the preceding command with `--post-snapshot-import true` fails, you should drop the INVALID indexes first using the following query - 

Reviewer: ```suggestion
If the preceding command with `--post-snapshot-import true` fails, you should drop the INVALID indexes first using the following query:
```

```sql
DO $$ 
DECLARE
  index_name text;
BEGIN
  FOR index_name IN (
    SELECT indexrelid::regclass 
    FROM pg_index 
    WHERE indisvalid = false
  ) 
  LOOP
    EXECUTE 'DROP INDEX ' || index_name;
  END LOOP;
END $$;
```

> retry the command with the argument `--ignore-exist` to ignore already created indexes and create new ones instead.

Reviewer: ```suggestion
Retry the command with the argument `--ignore-exist` to ignore already created indexes and create new ones instead.
```

    1. The [export data from target](../../reference/data-migration/export-data/#export-data-from-target) command automatically starts capturing changes from the target YugabyteDB database.
    Note that the [import data to target](#import-data-to-target) process transforms to an `export data from target` process, so if it gets terminated for any reason, you need to restart the process using the `export data from target` command as suggested in the `import data to target` output.

       {{<note title="Event duplication">}}

> The `export data from target` command may result in duplicated events if you restart Voyager, or there is a change in the YugabyteDB database server state. Consequently, the [get data-migration-report](#get-data-migration-report) command may display additional events that have been exported from the target YugabyteDB database, and imported into the source-replica or source database. For such situations, it is recommended to manually verify data in the target and source-replica, or source database to ensure accuracy and consistency.

Author: yes there's no source-replica in fall back.
